lib/utils/createConfig.js

```diff
@@ -88,6 +88,10 @@ function createConfig(config, argv, { port }) {
     options.hotOnly = argv.hotOnly;
   }
 
+  if (!options.liveReload) {
+    options.liveReload = argv.liveReload;
+  }
```

Comment:
https://github.com/webpack/webpack-dev-server/blob/9a86d02ce9b7ded3b274cdc4dcbc0a2ed85a8f2f/lib/utils/createConfig.js#L114 we should test `--no-live-reload` because be default `liveReload` should be `true`